forum/communications/serializers.py

```diff
@@ -1,10 +1,72 @@
 from rest_framework import serializers
+from .mongo_models import Room
+from .helpers import is_namespace_info_correct
+from bson.objectid import ObjectId
+from bson.errors import InvalidId
+
+
+class NamespaceInfoSerializer(serializers.Serializer):
+    user_id = serializers.IntegerField(required=True)
+    namespace = serializers.ChoiceField(choices=[
```

Comment:
You can use enum here too https://github.com/Project-Stage-Academy/UA1198ForumSB/blob/main/forum/communications/mongo_models.py#L15